cinch/playbooks/jenkins_restart_utility.yml

```diff
@@ -5,9 +5,16 @@
 
 # Example:
 # ansible-playbook -u root --private-key /path/to/key
-# -i '10.0.183.169', jenkins_restart_utility.yml -e "restart_jenkins=true"
+# jenkins_restart_utility.yml
+# -e "restart_jenkins=true hostname=jenkins
+# domain=example.com"
+
+# Optionally include an arbitary role here
+# to run before the playbook is executed if needed"
+- hosts: "{{ hostname }}.{{ domain }}"
+  roles:
+  - role: "{{ dependency_role | default() }}"
```

Comment:
Maybe this name could be improved? "dependency_role" might be better mnemonically as "pre_restart_role_name" or something like that.